app/assets/javascripts/components/utils.js.coffee

```diff
@@ -132,3 +132,9 @@ class @Utils
       .fail (xhr, status, error) ->
         alert('Error: ' + error)
         callback()
+
+  @select2TagClickHandler: (e, elem) ->
+    if e.which == 1
```

Comment:
I still would like to treat the agent links like every other link on the page. When you click on it by accident you can still navigate back without losing the changes made. In my tests the back button worked for both option editors with FF, Chrome and Safari.